python/mlcroissant/mlcroissant/_src/core/rdf.py

```diff
@@ -71,6 +72,14 @@ class Rdf:
     def from_json(cls, ctx, json: Json) -> Rdf:
         """Creates a `Rdf` from JSON."""
         context = get_context(json)
+        difference = context.keys() - make_context(ctx).keys()
```

Comment:
This would show only keys that are in the user defined's context and not in the standard context -- but not keys which are missing from the standard context, right?

Reply:
Done.

Good catch, I probably want to do the contrary.